ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -91,7 +91,7 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
     }
     assert(actualCode == expectedCode, s"Expected code [$expectedCode], but got [$actualCode].")
     assert(
-      message.contains(pattern),
+      Option(message).getOrElse("").contains(pattern),
       s"Error message did not contain [$pattern], but was [$message].")
```

Comment:
If it's nullable, it's probably best to handle this in the output as well so that it's the least confusing for the reader.